### DISCUSSION TITLE
How to create empty Detections object?

### DISCUSSION THREAD

**Discussion by adrianboguszewski** [roboflow/supervision#48] [Q&A]
How to deal with the situation when nothing is detected? Currently, there is no possibility to create a Detections object with empty arrays.

**Comment by SkalskiP** [comment]
Hi, @adrianboguszewski 👋🏻! Sorry for the late reply. I was on a short vacation. Currently, it is only possible to create empty detection like this:

```python
Detections(
    xyxy=np.empty((0, 4), dtype=np.float32),
    confidence=np.array([], dtype=np.float32),
    class_id=np.array([], dtype=int)
)
```

Would you be satisfied with some code sugar like this: `Detection.empty()` that would essentially do the same?

**Comment by adrianboguszewski** [comment]
Thanks for the answer. It wasn't so late :) 

Yeah, I think `Detections.empty()` (or similar) is worth considering.

**Reply by SkalskiP** [reply]
Okay I think you are right. I’ll keep you posted.
